src/pages/settings/expense-categories/Create.tsx

```diff
@@ -56,6 +60,8 @@ export function Create() {
           toast.dismiss();
           toast.success(t('created_expense_category'));
```

Comment:
@beganovich I did not change any part of the previously implemented logic for creating the expense category. We use the formik and the toaster directly from the package. Please let me know if I should include that part of the refactoring in this PR.